src/uu/cp/src/cp.rs

```diff
@@ -873,7 +888,7 @@ fn copy_source(
 }
 
 #[cfg(target_os = "windows")]
-fn adjust_canonicalization(p: &Path) -> Cow<Path> {
+fn adjust_canonicalization<'a>(p: &'a Path) -> Cow<'a, Path> {
```

Comment:
```suggestion
fn adjust_canonicalization(p: &Path) -> Cow<Path> {
```

`cargo clippy` says the explicit lifetimes are unneeded. My test compile `cargo +1.32.0 test` compiled and tested correctly.